Userland/Libraries/LibWeb/WebIDL/ExceptionOr.h

```diff
@@ -102,6 +102,11 @@ class [[nodiscard]] ExceptionOr {
         return move(m_result_or_exception.template get<ValueType>());
     }
 
+    ValueType release_value() const
```

Comment:
This doesn't make much sense. It will violate const-correctness of the ExceptionOr if you yoink the internals out of a const object.

Reply:
I got this error without the change, do you know an alternative solution?
```
IDBFactory.cpp:81:37: error: 'this' argument to member function 'release_value' has type 'const WebIDL::ExceptionOr<JS::NonnullGCPtr<IDBDatabase>>', but function is not marked const
```